components/experience-item/ExperienceItem.tsx

```diff
@@ -31,27 +31,33 @@ export const ExperienceItem = ({
       <FieldDisplay
         label={t('labels.position')}
         value={t(`profession.${experience.lineOfWork}-${gender}`)}
+        enabled={Boolean(experience.lineOfWork)}
```

Comment:
would be nice if we wouldn't have to repeat the value in the enabled field each time. Like checking for the value in the component or so.

But not necessary if this takes too much time / or is not possible

Reply:
The issue is that what we're checking for isn't always the whole value passed into the component. That's why I thought it would be most explicit to have an optional enabled property (which is set to true if nothing is passed in). I guess one could have the component check for the value AND an optional enabled property and only display stuff if both are truthy. I was first going to write that that's a bit too complicated and not so straightforward for my taste, but reading the description I quite like this option. Thanks!

Reply:
It's done now. Minor change and I like it better, too!